src/module.ts

```diff
@@ -25,13 +28,26 @@ export default defineNuxtModule<ModuleOptions>({
     siteKey: nuxt.options.dev ? '1x00000000000000000000AA' : undefined,
     addValidateEndpoint: false,
   }),
-  async setup (options, nuxt) {
+  async setup(options, nuxt) {
     const siteKey = options.siteKey || nuxt.options.runtimeConfig.public?.turnstile?.siteKey
     if (!siteKey) {
       console.warn('`@nuxtjs/turnstile` is disabled as no site key was provided.')
       return
     }
 
+    if (options.secretKeyPath) {
+      const secretKeyPath = options.secretKeyPath
+      const secretKeyFromPath =
+        secretKeyPath && fs.existsSync(secretKeyPath)
+          ? fs.readFileSync(secretKeyPath, 'utf-8')
+          : undefined
+
+      if (!secretKeyFromPath) {
+        console.warn('The secret key from the path could not be found.')
+      }
+      options.secretKey = secretKeyFromPath || options.secretKey
```

Comment:
As with other `if`s ;)

```suggestion
      }

      options.secretKey = secretKeyFromPath || options.secretKey
```